src/index.html

```diff
@@ -1,15 +1,211 @@
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="12"
+            max="100"
+            value="12"
+            class="field"
+            id="age"
+            required
+          >
+        </label>
+
+        <label for="date_of_birth" class="form__field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date of birth"
+            class="field"
+            id="date_of_birth"
+            required
+          >
+        </label>
+
+        <label for="term" class="form__field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            class="field"
+            id="term"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="form__title">Registration</legend>
+
+        <label for="email" class="form__field">
+          E-mail:
+          <input
+            type="email"
+            name="e-mail"
+            class="field"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label for="password" class="form__field">
+          Password:
+          <input
+            type="password"
+            name="Password"
+            class="field"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="form__title">An interesting fact about you!</legend>
+
+        <lablel for="love_cats_yes" class="form__field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="Love cats?"
+            value="yes"
+            id="love_cats_yes"
+            checked
+          >
+          <label for="love_cats_no">Yes</label>
+
+          <input
+            type="radio"
+            name="Love cats?"
+            value="no"
+            id="love_cats_no"
+          >
+          <label for="love_cats_no">No</label>
+        </lablel>
+
+        <label for="favorite_color" class="form__field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favorite color"
+            class="field"
+            id="favorite_color"
+          >
+        </label>
+
+        <label for="time_go_to_bed" class="form__field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time go to bed"
+            class="field"
+            id="time_go_to_bed"
+          >
+        </label>
+
+        <label for="cars" class="form__field">
+          What are your favorite brand of cars?
+          <select
+            name="favorite cars"
+            multiple
+            id="cars"
+          >
+            <option value="bmv">BMV</option>
+            <option value="audu">Audi</option>
```

Comment:
```suggestion
            <option value="audi">Audi</option>
```
Pls be more attentive